### ISSUE TITLE
404 on wia aria link

### ISSUE THREAD

**Issue by stephenmathieson** [joaovieira/tslint-react-a11y#2]
Just saw the error:

```
ERROR: [...]/src/app/components/RadioGroup.tsx:15:9 - Tag 'input' has implicit role 'radio'. It requires aria-* attributes: aria-checked that are missing in the element. A reference to role definitions can be found at https://www.w3.org/TR/wai-aria/roles#role_definitions.
```

Which is great, but the link (https://www.w3.org/TR/wai-aria/roles#role_definitions) 404s.

**Comment by chadlavi-casebook** [comment]
@joaovieira is this repo dead?

**Comment by joaovieira** [comment]
I have since migrated to `typescript-eslint`. But will look into this. Hopefully it's just a matter of updating dependencies.

**Comment by joaovieira** [comment]
Fixed in `tslint-react-a11y@1.1.0`. Sorry for the delay!
